src/main/java/com/contrastsecurity/sdk/ContrastSDK.java

```diff
@@ -190,6 +197,314 @@ public Users getOrganizationUsers(String organizationId) throws IOException, Una
             IOUtils.closeQuietly(is);
         }
     }
+    /**
+     * Get user based on Org and Id
+     * @param organizationId the ID of the organization
+     * @param  userId User ID
+     * @return A User Object
+     * @throws UnauthorizedException if the Contrast account failed to authorize
+     * @throws IOException           if there was a communication problem
+     */
+    public User getOrganizationUser(String organizationId, String userId) throws IOException, UnauthorizedException {
+        InputStream is = null;
+        InputStreamReader reader = null;
+        try {
+            is = makeRequest(HttpMethod.GET, this.urlBuilder.getUserUrl(organizationId, userId));
+            reader = new InputStreamReader(is);
+            NestedUser user = this.gson.fromJson(reader, NestedUser.class);
+            return user.getUser();
+        } finally {
+            IOUtils.closeQuietly(reader);
+            IOUtils.closeQuietly(is);
+        }
+    }
+
+    /**
+     * ddooley
+     * Create a User in an organization.
+     * Unfortunately can't reuse makeRequest without at least making changes there specific to new POST/PUT requests
+     * ToDo capture commonality into new methods as progress
+     * @param organizationId the ID of the organization
+     * @param userId the ID of the User
+     * @param firstName the User's first name
+     * @param lastName the User's last name
+     * @param orgRole the ID of the Org Role for the User
+     *                1 Admin
+     *                2 Rules Admin
+     *                3 Edit
+     *                4 View
+     * @param groupIds A list of group Ids. The User will be added to the respective groups
+     *
+     * @return Return HTTP Code.  200 is a success.
+     * @throws UnauthorizedException if the Contrast account failed to authorize
+     * @throws IOException           if there was a communication problem
+     */
+    public int createUser(String organizationId, String userId, String firstName, String lastName, long orgRole, ArrayList<Long> groupIds) throws IOException, UnauthorizedException {
+        HttpURLConnection connection = null;
+        try {
+            String url = restApiURL + this.urlBuilder.createUsersUrl(organizationId);
+            /* Create Connection */
+            connection = (HttpURLConnection) new URL(url).openConnection(this.proxy);

```

Comment:
This code block is duplicated a bunch in these additions. Extract it into something re-usable or make changes to address why we can't re-use makeRequest